File: codes/python/Code_3_04_1.py
# 入力
N = int(input())
B = list(map(int, input().split()))
R = list(map(int, input().split()))

# 答えの計算（Python 3 では B[i] / N, R[i] / N は小数まで計算されることに注意）
blue = 0.0
red = 0.0
for i in range(N):
	blue += B[i] / N
	red += R[i] / N

# 出力
print("%.12f" % (blue + red))

File: codes/c/Code_3_04_1.c
#include <stdio.h>

int N, B[100009], R[100009];

int main() {
	// 入力
	scanf("%d", &N);
	int i;
	for (i = 1; i <= N; i++) {
		scanf("%d", &B[i]);
	}
	for (i = 1; i <= N; i++) {
		scanf("%d", &R[i]);
	}

	// 答えの計算 → 答えの出力
	double blue = 0.0, red = 0.0;
	for (i = 1; i <= N; i++) {
		blue += 1.0 * B[i] / N;
		red += 1.0 * R[i] / N;
	}
	printf("%.12lf\n", blue + red);
	return 0;
}
